docker/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Read DOMAIN and ORIG_SPF
+. $HOME/.spf-toolsrc
+echo "$DOMAIN - $ORIG_SPF"
+echo "$(date)"
+$HOME/spf-tools/compare.sh $DOMAIN $ORIG_SPF || despf.sh $ORIG_SPF \
```

Comment:
@abtris I can see that the full path is only present when calling the first script; shouldn't you rather cd to this path before running all of them?